.rubocop.kapost.yml

```diff
@@ -33,21 +33,21 @@ Style/AndOr:
   EnforcedStyle: conditionals
 Style/BracesAroundHashParameters:
   EnforcedStyle: context_dependent
-Style/CaseIndentation:
+Layout/CaseIndentation:
   IndentOneStep: true
 Style/Documentation:
   Enabled: true
 Style/EachWithObject:
   Enabled: false
-Style/ExtraSpacing:
+Layout/ExtraSpacing:
```

Comment:
This is duplicated on L98.  Maybe we can combine and fix?

Reply:
Good catch. Combined.